crates/bevy_dynamic_plugin/src/loader.rs

```diff
@@ -1,7 +1,17 @@
 use libloading::{Library, Symbol};
+use thiserror::Error;
 
 use bevy_app::{App, CreatePlugin, Plugin};
 
+/// Errors that can occur when loading a dynamic plugin
+#[derive(Debug, Error)]
+pub enum DynamicPluginLoadError {
+    #[error("cannot load library for dynamic plugin: {0}")]
```

Comment:
Not sure what's the best practice here, but should we capitalize the first word?

```suggestion
    #[error("Cannot load library for dynamic plugin: {0}")]
```

Reply:
I say no, afaik it is common practice not to. (because it is common that errror messages might be printed/formatted embedded within a larger error message)

Also see https://github.com/bevyengine/bevy/blob/main/crates/bevy_asset/src/asset_server.rs#L19
Bevy does not capitalize the error messages in other places like the Asset Server. I followed the same style for consistency.

Reply:
Ok thanks for the explanation.